Signal/src/ViewControllers/ConversationView/Cells/OWSUnreadIndicatorCell.m

```diff
@@ -94,26 +97,14 @@ - (void)loadForDisplayWithTransaction:(YapDatabaseReadTransaction *)transaction
     self.titleLabel.text = [self titleForInteraction:interaction];
     self.subtitleLabel.text = [self subtitleForInteraction:interaction];
 
-    self.backgroundColor = [UIColor whiteColor];
+    self.subtitleLabel.hidden = self.subtitleLabel.text.length < 1;
 
     [NSLayoutConstraint deactivateConstraints:self.layoutConstraints];
     self.layoutConstraints = @[
-        [self.strokeView autoPinEdgeToSuperviewEdge:ALEdgeTop],
-        [self.strokeView autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
-        [self.strokeView autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterTrailing],
-        [self.strokeView autoSetDimension:ALDimensionHeight toSize:self.strokeHeight],
-
-        [self.titleLabel autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.strokeView],
-        [self.titleLabel autoPinEdgeToSuperviewEdge:ALEdgeTop],
-        [self.titleLabel autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
-        [self.titleLabel autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterTrailing],
-
-        [self.subtitleLabel autoPinEdge:ALEdgeTop
-                                 toEdge:ALEdgeBottom
-                                 ofView:self.titleLabel
-                             withOffset:self.subtitleVSpacing],
-        [self.subtitleLabel autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
-        [self.subtitleLabel autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterTrailing],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeTop],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeBottom],
+        [self.stackView autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.fullWidthGutterLeading],
```

Comment:
For full width stuff, don't we want to pin to the edge here? not the margin?

Reply:
I was counting on `autoPinLeadingToSuperviewMarginWithInset ` being equivalent `autoPinEdgeToSuperviewEdge:ALEdgeLeading withInset:` since the cell margins are empty.  I preferred the former because it's more concise, but I've switched for clarity.